index.js

```diff
@@ -92,7 +129,18 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+if(movies.length === 0){
+  throw "there are no movies"
+}
+  let movieObj = movies.filter((movie ,genre = []) => { 
```

Comment:
```suggestion
  let movieObj = movies.filter((movie) => { 
```
you don't need to set a genre parameter here! it's already being set on 132. One of the reasons this function isn't passing is because genre is always being set to and empty array --> []